### PR TITLE
Chapter 3

### PR DESCRIPTION
**Pull request by aisharina99**
ready

### CODE REVIEW

**Comment by aisharina99** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```
Comment (thaodtd-0665) [inline app/views/layouts/application.html.erb:4]: Research and apply "i18n" into project for translate string (using display on UI for end user)
https://guides.rubyonrails.org/i18n.html#providing-translations-for-internationalized-strings

```suggestion
    <title>t("title")</title>
```
```
# config/locales/en.yml
en:
  title: Sample App
```

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (thaodtd-0665) [inline config/database.yml:1]: 1. Config this file into .gitignore then run bellow command line --> do not push this file to pull request
`git rm --cached config/database.yml `
2. Create a new file with name "config/database.yml.example" --> this file using push to pull request

**Comment by thaodtd-0665** [comment]
Your pull have 2 commit, recommend rebase into 1 commit/pull request
https://stackoverflow.com/questions/2563632/how-can-i-merge-two-commits-into-one-if-i-already-started-rebase

**Comment by aisharina99** [comment]
ready

File: config/database.yml (hunk 2 above)
Comment (aisharina99) [reply config/database.yml:1]: ready

**Comment by ngocvt-0484** [comment]
not fix yet
please change to 1 commit

**Comment by ngocvt-0484** [comment]
commented

**Pull request closed without merging**